src/Metadata/schema/metadata.xsd

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+
+<xsd:schema
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+        attributeFormDefault="unqualified"
+        elementFormDefault="qualified">
+
+    <xsd:element name="resources" type="resources"/>
+
+    <xsd:complexType name="resources">
+            <xsd:sequence minOccurs="0" maxOccurs="unbounded">
+                <xsd:element name="resource" type="resource"/>
+            </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="resource">
+            <xsd:sequence minOccurs="0" maxOccurs="unbounded">
+                <xsd:element name="itemOperation" minOccurs="0" maxOccurs="unbounded" type="attribute"/>
+                <xsd:element name="collectionOperation" minOccurs="0" maxOccurs="unbounded" type="attribute"/>
+                <xsd:element name="attribute" minOccurs="0" maxOccurs="unbounded" type="attribute"/>
```

Comment:
No because you can add attributes directly inside the `resource` node:

```xml
<resources>
    <resource class="ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\Dummy"/>
        <attribute name="normalization_context">
            <attribute name="groups">
                <attribute>default</attribute>
            </attribute>
        </attribute>
        <attribute name="denormalization_context">
            <attribute name="groups">
                <attribute>default</attribute>
            </attribute>
        </attribute>
</resource>
```